web-client/src/components/WebClientMap/WebClientMap.tsx

```diff
@@ -52,6 +52,11 @@ const WebClientMap: React.FC<MapProps> = ({
   const initGoogleMapServices = ({ map, maps }) => {
     if (map && maps) {
       googleMap || setGoogleMap(map);
+      /*
+      map.controls[google.maps.ControlPosition.RIGHT_CENTER].push(
+        <div style={{ zIndex: 1000 }}>hello world</div>,
+      );
+      */
```

Comment:
I'm guessing this is some accidental left over code?

Reply:
removed